manifest/yaml.go

```diff
@@ -280,6 +280,9 @@ func (l *Labels) UnmarshalYAML(unmarshal func(interface{}) error) error {
 
 			switch len(parts) {
 			case 2:
+				if dup, ok := (*l)[parts[0]]; ok {
+					return fmt.Errorf("duplicate cron label %v", dup)
```

Comment:
Could we use `parts[0]` (the key) instead of `dup`? 

`duplicate cron label convox.cron.myjob` seems easier to read than `duplicate cron label * * * * ? bin/myjob1`